README.md

```diff
@@ -325,11 +325,18 @@ If you need access to the full dataset, you can read the event files that
 TensorBoard consumes by using the [`summary_iterator`](https://github.com/tensorflow/tensorflow/blob/e7f333b5f8b3c53b21d149d8d14c0cebbde431aa/tensorflow/python/summary/summary_iterator.py#L313)
 method.
 
+### Can I customize which lines appear in a plot?
 
-### Can I overlap multiple plots?
+Using the [custom scalars plugin](plugins/custom_scalar), you can create 
+scalar plots with lines for custom run-tag pairs. However, within the original
+scalars dashboard, each scalar plot contains lines for a single tag: Each line
```

Comment:
Optional wording suggestion: "each scalar plot corresponds to data for a specific tag and contains lines for each run that includes that tag."

Reply:
That seems easier to read. Done.